tests/settings.py

```diff
@@ -64,7 +64,7 @@
     'LONG_VALUE': (long_value, 'some looong int'),
     'BOOL_VALUE': (True, 'true or false'),
     'STRING_VALUE': ('Hello world', 'greetings'),
-    'UNICODE_VALUE': (six.u('Rivière-Bonjour'), 'greetings'),
+    'UNICODE_VALUE': (u'Rivière-Bonjour', 'greetings'),
```

Comment:
https://pythonhosted.org/six/#six.u

> On Python 2, u() doesn’t know what the encoding of the literal is. Each byte is converted directly to the unicode codepoint of the same value. Because of this, it’s only safe to use u() with strings of ASCII data.

So if we do need to support Python 3.2 in tests we can us six.u(), but with escape sequences instead of the accented characters.